Resources/doc/3-4-format_listener.rst

```diff
@@ -0,0 +1,129 @@
+3.4 Format listener
+===================
+
+The Request format listener attempts to determine the best format for the
+request based on the Request's Accept-Header and the format priority
```

Comment:
`for the request based on the Request's Accept-Header` looks confusing to me. What about rewording it as:

```
for the request based on the HTTP Accept header
```